api/runtime/hooks/v1alpha1/inplaceupdate_types.go

```diff
@@ -34,37 +34,26 @@ type CanUpdateMachineRequest struct {
 
 	// current contains the current state of the Machine and related objects.
 	// +required
-	Current CanUpdateMachineRequestObjects `json:"current"`
+	Current CanUpdateMachineRequestObjects `json:"current,omitempty,omitzero"`
```

Comment:
API conventions currently recommend to use omitempty + omitzero on all "struct fields" (doesn't matter if optional or required)